wwwroot/inc/snmp.php

```diff
@@ -4436,6 +4454,30 @@
 		'text' => ' J9049A: 24 RJ-45/10-100-1000T',
 		'processors' => array ('procurve-21-to-24-combo-1000SFP', 'procurve-chassis-1000T'),
 	),
+	'14988.2' => array
```

Comment:
`14988.2` appears to be the root OID for all Mikrotik switches.  it would be safer to handle this in the doSwitchSNMPMining() function, otherwise all Mikrotik switches will get identified as this particular model.

i.e. if `14988.2`, examine some other OID to determine the specific switch model.  you can also add other ports from there, such as console & AC